pkg/uhttp/gocache.go

```diff
@@ -0,0 +1,129 @@
+package uhttp
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"io"
+	"net/http"
+	"net/http/httputil"
+	"time"
+
+	bigcache "github.com/allegro/bigcache/v3"
+)
+
+type GoCache struct {
+	ttl         time.Duration
+	rootLibrary *bigcache.BigCache
```

Comment:
It'd be nice to have some stats in this struct so we could keep track of cache hit/miss rates.

Reply:
Actually it looks like bigcache has its own stats function that we can call: https://pkg.go.dev/github.com/allegro/bigcache#BigCache.Stats

Reply:
Fixed